graphql-schemas/schema.yml

```diff
@@ -1886,6 +1886,13 @@ confs:
   - { name: openshiftResources, type: NamespaceOpenshiftResource_v1, isList: true, isInterface: true, isRequired: true }
   - { name: openshiftServiceAccountTokens, type: ServiceAccountTokenSpec_v1, isList: true }
   - { name: glitchtipProjects, type: GlitchtipProjects_v1, isList: true }
+  - name: namespaces
```

Comment:
Why do you need a backref to namespaces?

Reply:
to understand later where to put Secrets with the DSN